src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -489,6 +510,17 @@ exprt gdb_value_extractort::get_pointer_value(
       return result_expr;
     }
 
+    // pointer to function
+    if(expr.type().subtype().id() == ID_code)
+    {
+      const auto target_expr =
+        get_pointer_to_function_value(expr, value, location);
+      CHECK_RETURN(target_expr.id() != ID_nil);
+      const auto result_expr = address_of_exprt(target_expr);
```

Comment:
I'd use either `exprt result_expr = ...` or `address_of_exprt result_expr{target_expr};` (see below for more).

Reply:
I went for the `address_of_exprt` and then `std::move`.